media/js/mozorg/home/takeover-2016.js

```diff
@@ -67,18 +70,21 @@
         $options.removeAttr('checked');
     }
 
-    // Shows the take over modal, sets the required entry in
-    // session storage and adds an `esc` key handler to close
-    // the modal.
+    // Shows the take over modal, sets the seven day cookie
+    // and adds an `esc` key handler to close the modal.
     function initTakeOver() {
         // format the currency in browsers that support it
         if (intlNumberFormatSupported) {
             formatCurrency();
         }
-        // show the takeover
-        toggleModal();
-        // set mozorg.takeover.seen to true
-        storeSession();
+
+        // Don't show the takeover if cookies are disabled or not supported
+        if (document.cookie) {
```

Comment:
Hmm, actually I'm not sure if `document.cookie` is enough here, would this return a false positive if no cookies exist (irrespective of being enabled or not). Perhaps simply `Mozilla.Cookies.hasItem('eoy-takeover')` would be enough?